### DISCUSSION TITLE
Unable to enforce HTTP github pages

### DISCUSSION THREAD

**Discussion by Daniel-Agoston** [community/community#42730] [Other Feature Feedback, Questions, & Ideas]
Github Pages

![Capture](https://user-images.githubusercontent.com/91429039/209731446-7a2bcad5-c2f5-4be7-a6ef-f76a6be085ab.PNG)

Hello,

My profile: https://github.com/Daniel-Agoston

domain concerned: danielagoston.com

I am trying to configure my sites domain, but can't enforce the http. I get all sorts of intermittent error messages, and also one stating DNS check successful (although now it's saying "DNS valid for primary"). I've included a screen shot of how I've configured the DNS with GoDaddy so far, but can't get my domain to work. Can anyone see what's wrong please?

The error message I am getting now is:

"Both danielagoston.com and its alternate name are improperly configured
Domain does not resolve to the GitHub Pages server. (NotServedByPagesError)."

and

"www.danielagoston.com is improperly configured
Your site's DNS settings are using a custom subdomain, www.danielagoston.com, that is not set up with a correct CNAME record. We recommend you change this to a CNAME record pointing to daniel-agoston.github.io. (InvalidCNAMEError)."

I've tried everything, looked up online, different forums, videos but nothing seems to work.

Thanks a lot and happy holidays!

Dan

**Comment by yoannchaudet** [comment]
Remove the fifth `A` record labelled "Parked". This is an additional DNS record that does not point to GitHub Pages and as such won't allow us to obtain a TLS certificate on your behalf.

![image](https://user-images.githubusercontent.com/14911070/209738582-6c41dfe4-6faa-4a3e-8a44-8589569a70de.png)

**Reply by Daniel-Agoston** [reply]
Thank you very much Yoann, it worked!! Happy holidays!!

**Reply by nasifun** [reply]
I need help, it has been 5 days since i m using GITHUB, was highly recommended. Every thing was quite fine, till I reached a stage connecting repo DNS to my domain.  it did work at all i have a screen shoot. Anyone with a solution,
![DNS IN GITHUB NOT WORKING](https://github.com/user-attachments/assets/68078397-4a85-43f7-8197-2629897ce84d)
 please help out
